src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;
+
+public class FruitTransaction {

```

Comment:
The class `FruitTransaction` should be documented with a Javadoc comment explaining its purpose, especially since it seems to be a core part of your application domain.